src/main/kotlin/app/revanced/patches/music/misc/microg/patch/resource/MusicMicroGResourcePatch.kt

```diff
@@ -0,0 +1,44 @@
+package app.revanced.patches.music.misc.microg.patch.resource

```

Comment:
This file seems also very similar.

Reply:
dont know how to do both currently but will figure it out a way 👍